src/components/card/card.tsx

```diff
@@ -52,7 +52,7 @@ const CardThumbnail = ({
 
     const defaultThumbnail = (
         <Image
-            alt={alt || 'alt not provided'}
+            alt={alt || ''}
```

Comment:
Also a bug fix, having this displayed as an alt feels weird from a user's POV.

Reply:
What about a placeholder of just like "MongoDB image" or something?